quarry/data/packets/0736_1.16.1.csv

```diff
@@ -0,0 +1,153 @@
+protocol_mode,packet_direction,packet_name
+init,upstream,handshake
+play,downstream,spawn_object
+play,downstream,spawn_experience_orb
+play,downstream,spawn_global_entity
+play,downstream,spawn_mob
+play,downstream,spawn_painting
+play,downstream,spawn_player
+play,downstream,animation
+play,downstream,statistics
+play,downstream,acknowledge_player_digging
+play,downstream,block_break_animation
+play,downstream,update_block_entity
+play,downstream,block_action
+play,downstream,block_change
+play,downstream,boss_bar
+play,downstream,server_difficulty
+play,downstream,chat_message
+play,downstream,multi_block_change
+play,downstream,tab_complete
+play,downstream,declare_commands
+play,downstream,confirm_transaction
+play,downstream,close_window
+play,downstream,window_items
+play,downstream,window_property
+play,downstream,set_slot
+play,downstream,set_cooldown
+play,downstream,plugin_message
+play,downstream,named_sound_effect
+play,downstream,disconnect
+play,downstream,entity_status
+play,downstream,explosion
+play,downstream,unload_chunk
+play,downstream,change_game_state
+play,downstream,open_horse_window
+play,downstream,keep_alive
+play,downstream,chunk_data
+play,downstream,effect
+play,downstream,particle
+play,downstream,update_light
+play,downstream,join_game
+play,downstream,map
+play,downstream,trade_list
+play,downstream,entity_relative_move
+play,downstream,entity_look_and_relative_move
+play,downstream,entity_look
+play,downstream,entity
+play,downstream,vehicle_move
+play,downstream,open_book
+play,downstream,open_window
+play,downstream,open_sign_editor
+play,downstream,craft_recipe_response
+play,downstream,player_abilities
+play,downstream,combat_event
+play,downstream,player_list_item
+play,downstream,face_player
+play,downstream,player_position_and_look
+play,downstream,unlock_recipes
+play,downstream,destroy_entities
+play,downstream,remove_entity_effect
+play,downstream,resource_pack_send
+play,downstream,respawn
+play,downstream,entity_head_look
+play,downstream,select_advancement_tab
+play,downstream,world_border
+play,downstream,camera
+play,downstream,held_item_change
+play,downstream,update_view_position
+play,downstream,update_view_distance
+play,downstream,display_scoreboard
+play,downstream,entity_metadata
+play,downstream,attach_entity
+play,downstream,entity_velocity
+play,downstream,entity_equipment
+play,downstream,set_experience
+play,downstream,update_health
+play,downstream,scoreboard_objective
+play,downstream,set_passengers
+play,downstream,teams
+play,downstream,update_score
+play,downstream,spawn_position
```

Comment:
The spawn position packet was moved to 0x42 (below update_view_distance). This needs to be updated here to prevent naming issues.